qualtran/bloqs/block_encoding/chebyshev_polynomial.py

```diff
@@ -162,11 +160,10 @@ def build_call_graph(self, ssa: 'SympySymbolAllocator') -> Set['BloqCountT']:
 
 @bloq_example
 def _chebyshev_poly_even() -> ChebyshevPolynomial:
-    from qualtran.bloqs.basic_gates import Hadamard, XGate
-    from qualtran.bloqs.block_encoding import LinearCombination, Unitary
+    from qualtran.bloqs.basic_gates import XGate
+    from qualtran.bloqs.block_encoding import Unitary
 
-    bloq = LinearCombination((Unitary(XGate()), Unitary(Hadamard())), (0.5, 0.5), lambd_bits=1)
```

Comment:
isn't $X + H$ is also hermitian?

Reply:
Nice catch. It isn't, but $(X+H)/\sqrt{2+\sqrt{2}}$ is. That makes it suitable for a test for `ScaledChebyshevPolynomial` with $\alpha = \sqrt{2+\sqrt{2}}$, and I have made it one. It won't work for `ChebyshevPolynomial` though.